nnvm/python/nnvm/compiler/build_module.py

```diff
@@ -299,7 +299,8 @@ def build(graph, target=None, shape=None, dtype="float32",
             graph._set_json_attr("opt_level", 0, "int")
         graph = graph.apply("InferShape").apply("InferType")
         with target:
-            graph = graph.apply("GraphFusePartition").apply("GraphFuseCompile")
+            graph = graph.apply("GraphFusePartition").apply(
+                "GraphFuse").apply("GraphCompile")
```

Comment:
I'd suggest move 
```
  graph = graph.apply("GraphFusePartition").apply(
                "GraphFuse")
```
to outside of with target above, as fusion should be target independent.

And separate it into two lines,
```
graph = graph.apply("GraphFusePartition")
graph = graph.apply("GraphFuse")
```
because in the future we want to add other fusion rules between "GraphFusePartition" and "GraphFuse".

I think "GraphFusePartition" should be renamed to something like "GraphFindFusibleGroups". It makes the separation of work between "Partition step" and "Fusion step" clearer. What do you think?